src/components/map/controls/layer-toggle/component.jsx

```diff
@@ -1,26 +1,46 @@
 import React from 'react';
-import './styles.scss';
+import ReactTooltip from 'react-tooltip';
+import PropTypes from 'prop-types';
 
-function LayerToggle({layers, setLayers}) {
+import Icon from 'components/icon';
+import LayersToggleModal from './modal';
+
+import styles from './styles.module.scss';
+
+const LayerToggleComponent = ({ tooltipRef, toggleLayerActive, setToggleLayerActive }) => {
   return (
-    <div className="c-layer-toggle">
-      {layers && layers.map((l, i) =>
-        <p key={`layer-${i}`}>
-          Layer {i+1}
-          <input
-            type="checkbox"
-            checked={l.active}
-            onChange={() =>
-              setLayers(layers.map(layer =>
-                layer.id !== l.id ?
-                layer :
-                {...l, active: !l.active}
-              ))
-            }></input>
-        </p>
-      )}
+    <div className={styles.layerToggle}>
+      <button
+        data-for="layerTootlip"
+        className={styles.mapNavigationButton}
+        data-tip=""
+        style={{ filter: toggleLayerActive ? 'invert(1)' : 'unset' }}
+      >
+        <Icon name="icon-stack" />
+      </button>
+      <ReactTooltip
+        id="layerTootlip"
+        ref={tooltipRef}
+        clickable
+        className="layerTooltip"
+        effect="solid"
+        globalEventOff="click"
+        event="click"
+        place="right"
+        afterShow={() => setToggleLayerActive(true)}
+        afterHide={() => setToggleLayerActive(false)}
+        offset={{ right: 1, bottom: 98 }} // bottom: 240px/2 - 45px/2 = 98px <=> (modal height / 2) - (toggle button height / 2), right: 1 <=> avoid overlapping modal with button border
```

Comment:
I meant something along the lines: https://til.hashrocket.com/posts/sxbrscjuqu-share-scss-variables-with-javascript
I don't think we need to add anything to webpack as this should be working by default, could you check? if it doesn't work then I would drop it.